DQM/Integration/python/clients/beamhltfake_dqm_sourceclient-live_cfg.py

```diff
@@ -106,84 +69,62 @@
 from DQM.Integration.config.online_customizations_cfi import *
 process = customise(process)
 
-#--------------------------
-# Proton-Proton Stuff
-#--------------------------
+process.dqmBeamMonitor.monitorName = 'TrackingHLTBeamspotStream'
+process.dqmBeamMonitor.OnlineMode = True              
+process.dqmBeamMonitor.recordName = BSOnlineRecordName
+
+process.dqmBeamMonitor.resetEveryNLumi   = 5
+process.dqmBeamMonitor.resetPVEveryNLumi = 5
+
+#---------
+# Upload BeamSpotOnlineObject (HLTRcd) to CondDB
+if unitTest == False:
+  process.OnlineDBOutputService = cms.Service("OnlineDBOutputService",
+
+      DBParameters = cms.PSet(
+                              messageLevel = cms.untracked.int32(0),
+                              authenticationPath = cms.untracked.string('')
```

Comment:
It looks like the `.` is missing here as the location if `db.key` is accessed relative to the current directory of the process.

Reply:
sorry I do not understand if I need to put it back the "." or not. I am confused as on lxplus it was working fine int he test I made so I did not know it was wrong.

Reply:
When there's no dot, the path is interpreted as an absolute path. 

The DB file in question is located here on our P5 machine: `/home/dqmdevlocal/output/.cms_cond/db.key`
The working directory of all of our Online clients is this one: `/home/dqmdevlocal/output/`
It appears that you reference the DB file by concatenating the `authenticationPath` with the relative file path (`.cms_cond/db.key`): https://github.com/cms-sw/cmssw/blob/master/CondCore/CondDB/interface/DecodingKey.h#L34

So if there's no dot, the code attempts to search for a DB file here: `/.cms_cond/db.key` instead of in the correct place: `/home/dqmdevlocal/output/.cms_cond/db.key`

Reply:
The dot is needed so that the file is seeked locally to current directory
Thanks

Reply:
I will provide a new PR (and backport) to add the '.' and the runNumber (together with another needed fix to the beamspot clients). What about Simone's question [here](https://github.com/cms-sw/cmssw/pull/31965#discussion_r523052120) ?

Reply:
Thanks @francescobrivio 

The configuration for dqmSaverPB is the same as dqmSaver, please see:

https://github.com/cms-sw/cmssw/blob/56e25f2107db38fdb3e77049c5278af483f20140/DQM/Integration/python/config/environment_cfi.py#L81

Reply:
Thanks @jfernan2 
One question: should the configuration for `dqmSaverPB` be imported separately as you suggest or the line `process.load("DQM.Integration.config.environment_cfi")` is enough for both `dqmSaver` and `dqmSaverPB`?

Reply:
Yes, it is enough, both modules are defined there
Thanks